### PR TITLE
virsh_cpu_models: Change the order for remote test

### PR DESCRIPTION
**Pull request by cliping**
Local host test don't need set remote_ip and remote_pwd. So change the code order for remote test.

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

#avocado run --vt-type libvirt --vt-machine-type i440fx virsh.cpu_models
JOB ID     : d3fed66d5e49367e84cae3af8aaedd244cf88efb
JOB LOG    : /root/avocado/job-results/job-2018-01-10T23.00-d3fed66/job.log
 (1/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.local_host.auto_get_arch: PASS (1.64 s)
 (2/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.local_host.specific_arch: PASS (1.53 s)
 (3/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.remote_host.auto_get_arch: PASS (6.56 s)
 (4/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.positive_test.remote_host.specific_arch: PASS (6.08 s)
 (5/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.negative_test.invalid_arch: PASS (1.53 s)
 (6/6) type_specific.io-github-autotest-libvirt.virsh.cpu_models.negative_test.invalid_option: PASS (1.56 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 21.81 s

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 1]
```diff
@@ -15,17 +15,16 @@ def run(test, params, env):
...
 
-    if 'EXAMPLE.COM' in remote_ip:
-        test.error("Please replace '%s' with valid remote_ip" % remote_ip)
-
     if remote_ref == "remote":
+        remote_ip = params.get("remote_ip", "REMOTE.EXAMPLE.COM")
+        remote_pwd = params.get("remote_pwd", None)
+
+        if 'EXAMPLE.COM' in remote_ip:
+            test.error("Please replace '%s' with valid remote_ip" % remote_ip)
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: test.cancel()

Comment (lento-sun) [reply libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: yes, pls change to test.cancel to follow other cases' convention.

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 2]
```diff
@@ -15,17 +15,16 @@ def run(test, params, env):
...
     remote_ref = params.get("remote_ref", "")
-
-    remote_ip = params.get("remote_ip", "REMOTE.EXAMPLE.COM")
-    remote_pwd = params.get("remote_pwd", None)
-
     connect_uri = libvirt_vm.normalize_connect_uri(params.get("connect_uri",
                                                               "default"))
 
-    if 'EXAMPLE.COM' in remote_ip:
-        test.error("Please replace '%s' with valid remote_ip" % remote_ip)
```
Comment (lento-sun) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: pls change to test.cancel() to follow other cases' convention.

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 3]
```diff
@@ -15,17 +15,16 @@ def run(test, params, env):
...
-
     if remote_ref == "remote":
+        remote_ip = params.get("remote_ip", "REMOTE.EXAMPLE.COM")
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: "REMOTE.EXAMPLE.COM" is host name, so maybe remote_ip rename to remote_hostname

File: libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py [hunk 4]
```diff
@@ -15,17 +15,16 @@ def run(test, params, env):
...
+        remote_pwd = params.get("remote_pwd", None)
+
+        if 'EXAMPLE.COM' in remote_ip:
+            test.cancel("Please replace '%s' with valid remote_ip" % remote_ip)
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_cpu_models.py]: "Please replace '%s' with valid remote_ip" change to "Please replace '%s' with valid remote ip"

**Pull request merged**